Flagsmith.Engine/Segment/Evaluator.cs

```diff
@@ -53,16 +53,16 @@ static bool TraitsMatchSegmentCondition(List<TraitModel> identityTraits, Segment
 
         public static bool MatchesTraitValue(object traitValue, SegmentConditionModel condition)
         {
-            var exceptionOperatorMethods = new Dictionary<string, string>()
-            {
-                { Constants.NotContains, "EvaluateNotContains" },
-                { Constants.Regex, "EvaluateRegex" },
-                { Constants.Modulo, "EvaluateModulo" },
```

Comment:
I'd rather revert the formatting change here if possible. Not 100% sure if this is correct formatting for c# but it's currently consistent throughout the codebase so we should try to keep it that way (or submit a PR to change it everywhere if we think it's wrong).

Reply:
Sure, done.
Any tips for setting up auto-formatting to support the common style?

Reply:
Sorry, forgot to respond here. I don't have anything built in actually. I just run `dotnet format` manually I think... we should add this to a pre-commit hook.